Podfile

```diff
@@ -26,9 +26,28 @@ target 'Campus' do
 end
 
 post_install do |installer|
+    
+    targets_swift_4 = ['Kanna','TKSubmitTransition','SWXMLHash','CalendarLib','ASWeekSelectorView','PZPullToRefresh','AYSlidingPickerView','Fuzzi']
+    
     installer.pods_project.targets.each do |target|
-        target.build_configurations.each do |config|
-            config.build_settings['SWIFT_VERSION'] = '3.2'
+        if targets_swift_4.include? target.name
+            target.build_configurations.each do |config|
+                config.build_settings['SWIFT_VERSION'] = '4.0'
```

Comment:
these only defer on this line. Move the if here to remove duplicated code

Reply:
done